packages/agent-base/src/index.ts

```diff
@@ -99,6 +115,20 @@ export abstract class Agent extends http.Agent {
 			}, cb);
 	}
 
+	upgradeSocketToTls(
+		servername: string | undefined,
+		opts: tls.ConnectionOptions,
+		socket?: net.Socket,
+	): net.Socket {
```

Comment:
We can drop this too

```suggestion
	) {
```

Reply:
Done